schemas.yaml

```diff
@@ -574,6 +563,76 @@ schemas:
           ipAddress: 192.168.1.13
           subnetMask: 255.255.255.0
           standardGateway: 192.168.1.1
+
+  voltageObject:
+    type: object
+    properties:
+      value:
+        type: number
+        minimum: 0
+        example: 24
+      unit:
+        type: string
+        minLength: 1
+        maxLength: 32
+        example: "V"
+  currentObject:
+    type: object
+    properties:
+      value:
+        type: number
+        minimum: 0
+        example: 0.5
+      unit:
+        type: string
+        minLength: 1
+        maxLength: 32
+        example: "A"
```

Comment:
The "value" and "unit" properties can be mandatory.

Reply:
Please check the latest commit and resolve conservation if it fits your expectations.